src/Swarm/Game/Scenario/Cells.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE DerivingVia #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Swarm.Game.Scenario.Cells (
```

Comment:
I think we typically use singular nouns for module names (e.g. `Swarm.Game.Entity`, `Swarm.Game.Recipe`, `Swarm.Language.Context`, etc.).  I would prefer this to be named `Swarm.Game.Scenario.Cell`.